### PR TITLE
Support for LCA_Toolkit refactor

### PR DESCRIPTION
**Pull request by michaelhoehn**
### Issues addressed by this PR

Closes #

### CODE REVIEW

File: BHoMUpgrader42/Upgrades.json [hunk 1]
```diff
@@ -29,9 +29,25 @@
...
+      "BH.Engine.LifeCycleAssessment.Compute.EnvironmentalProductDeclarationMetrics.DepletionOfAbioticResources": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.Engine.LifeCycleAssessment.Compute.EnvironmentalProductDeclarationMetrics.GlobalWarmingPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.Engine.LifeCycleAssessment.Compute.EnvironmentalProductDeclarationMetrics.PhotochemicalOzoneCreationPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.Engine.LifeCycleAssessment.Query.GetElementDensity": "This method has been removed as GetEPDDensity now contains this functionality.",
+      "BH.oM.LifeCycleAssessment.MaterialFragment.SectorEnvironmentalProductDeclaration": "This object has been replaced with a single EPD object and a specific Type parameter.",
+      "BH.oM.LifeCycleAssessment.MaterialFragment.IEnvironmentalProductDeclaration": "This interface has been removed as changes to the EPD object have rendered this unnecessary.",
```
Comment (FraserGreenroyd) [inline BHoMUpgrader42/Upgrades.json:49]:
```suggestion
      "BH.Engine.LifeCycleAssessment.Compute.AcidificationPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
      "BH.Engine.LifeCycleAssessment.Compute.EutrophicationPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
      "BH.Engine.LifeCycleAssessment.Compute.DepletionOfAbioticResources": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
      "BH.Engine.LifeCycleAssessment.Compute.DepletionOfAbioticResourcesFossilFuels": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
      "BH.Engine.LifeCycleAssessment.Compute.GlobalWarmingPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
      "BH.Engine.LifeCycleAssessment.Compute.OzoneDepletionPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
      "BH.Engine.LifeCycleAssessment.Compute.PhotochemicalOzoneCreationPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality."
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]
I think the JSON needs to change to the suggested.

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by michaelhoehn** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by michaelhoehn** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

File: BHoMUpgrader42/Upgrades.json [hunk 2]
```diff
@@ -29,10 +29,16 @@
...
+      "BH.Engine.LifeCycleAssessment.Compute.EutrophicationPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.Engine.LifeCycleAssessment.Compute.DepletionOfAbioticResources": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.Engine.LifeCycleAssessment.Compute.DepletionOfAbioticResourcesFossilFuels": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.Engine.LifeCycleAssessment.Compute.GlobalWarmingPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.Engine.LifeCycleAssessment.Compute.OzoneDepletionPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.Engine.LifeCycleAssessment.Compute.PhotochemicalOzoneCreationPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality."
```
Comment (FraserGreenroyd) [inline BHoMUpgrader42/Upgrades.json:39]:
```suggestion
      "BH.Engine.LifeCycleAssessment.Compute.PhotochemicalOzoneCreationPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
      "BH.oM.LifeCycleAssessment.MaterialFragment.SectorEnvironmentalProductDeclaration": "This object has been replaced with a single EPD object and a specific Type parameter.",
      "BH.oM.LifeCycleAssessment.MaterialFragment.IEnvironmentalProductDeclaration": "This interface has been removed as changes to the EPD object have rendered this unnecessary."
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

File: BHoMUpgrader42/Upgrades.json [hunk 3]
```diff
@@ -29,10 +29,18 @@
...
+      "BH.Engine.LifeCycleAssessment.Compute.DepletionOfAbioticResourcesFossilFuels": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.Engine.LifeCycleAssessment.Compute.GlobalWarmingPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.Engine.LifeCycleAssessment.Compute.OzoneDepletionPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.Engine.LifeCycleAssessment.Compute.PhotochemicalOzoneCreationPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.oM.LifeCycleAssessment.MaterialFragment.SectorEnvironmentalProductDeclaration": "This object has been replaced with a single EPD object and a specific Type parameter.",
+      "BH.oM.LifeCycleAssessment.MaterialFragment.IEnvironmentalProductDeclaration": "This interface has been removed as changes to the EPD object have rendered this unnecessary."
```
Comment (FraserGreenroyd) [inline BHoMUpgrader42/Upgrades.json:41]:
```suggestion
      "BH.oM.LifeCycleAssessment.MaterialFragment.IEnvironmentalProductDeclarationData": "This interface has been removed as changes to the EPD object have rendered this unnecessary."
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

File: BHoMUpgrader42/Upgrades.json [hunk 4]
```diff
@@ -29,10 +29,18 @@
...
+      "BH.Engine.LifeCycleAssessment.Compute.DepletionOfAbioticResourcesFossilFuels": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.Engine.LifeCycleAssessment.Compute.GlobalWarmingPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.Engine.LifeCycleAssessment.Compute.OzoneDepletionPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.Engine.LifeCycleAssessment.Compute.PhotochemicalOzoneCreationPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+      "BH.oM.LifeCycleAssessment.MaterialFragment.SectorEnvironmentalProductDeclaration": "This object has been replaced with a single EPD object and a specific Type parameter.",
+      "BH.oM.LifeCycleAssessment.MaterialFragment.IEnvironmentalProductDeclarationData": "This interface has been removed as changes to the EPD object have rendered this unnecessary."
```
Comment (FraserGreenroyd) [inline BHoMUpgrader42/Upgrades.json:41]:
```suggestion
      "BH.oM.LifeCycleAssessment.MaterialFragments.SectorEnvironmentalProductDeclaration": "This object has been replaced with a single EPD object and a specific Type parameter.",
      "BH.oM.LifeCycleAssessment.MaterialFragments.IEnvironmentalProductDeclarationData": "This interface has been removed as changes to the EPD object have rendered this unnecessary."
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

File: BHoMUpgrader42/Upgrades.json [hunk 5]
```diff
@@ -29,10 +29,18 @@
 
     }
   },
-  "MessageForDeleted": {
-
-  },
+    "MessageForDeleted": {
+      "BH.Engine.LifeCycleAssessment.Compute.AcidificationPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```
Comment (FraserGreenroyd) [inline BHoMUpgrader42/Upgrades.json:33]:
```suggestion
      "BH.Engine.LifeCycleAssessment.Compute.AcidificationPotential(BH.oM.Dimensional.IElementM, BH.oM.LifeCycleAssessment.MaterialFragments.IEnvironmentalProductDeclarationData)": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
"BH.Engine.LifeCycleAssessment.Compute.AcidificationPotential(System.Double, System.Double, System.Double)", "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```

File: BHoMUpgrader42/Upgrades.json [hunk 6]
```diff
@@ -29,10 +29,18 @@
...
+      "BH.Engine.LifeCycleAssessment.Compute.EutrophicationPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```
Comment (FraserGreenroyd) [inline BHoMUpgrader42/Upgrades.json:34]:
```suggestion
      "BH.Engine.LifeCycleAssessment.Compute.EutrophicationPotential(System.Double, System.Double, System.Double)" : "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
"BH.Engine.LifeCycleAssessment.Compute.EutrophicationPotential(BH.oM.Dimensional.IElementM, BH.oM.LifeCycleAssessment.MaterialFragments.IEnvironmentalProductDeclarationData)" : "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```

File: BHoMUpgrader42/Upgrades.json [hunk 7]
```diff
@@ -29,10 +29,18 @@
...
+      "BH.Engine.LifeCycleAssessment.Compute.DepletionOfAbioticResources": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```
Comment (FraserGreenroyd) [inline BHoMUpgrader42/Upgrades.json:35]:
```suggestion
      "BH.Engine.LifeCycleAssessment.Compute.DepletionOfAbioticResources(System.Double, System.Double, System.Double)" : "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
"BH.Engine.LifeCycleAssessment.Compute.DepletionOfAbioticResources(BH.oM.Dimensional.IElementM, BH.oM.LifeCycleAssessment.MaterialFragments.IEnvironmentalProductDeclarationData)" : "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```

File: BHoMUpgrader42/Upgrades.json [hunk 8]
```diff
@@ -29,10 +29,18 @@
...
+      "BH.Engine.LifeCycleAssessment.Compute.DepletionOfAbioticResourcesFossilFuels": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```
Comment (FraserGreenroyd) [inline BHoMUpgrader42/Upgrades.json:36]:
```suggestion
      "BH.Engine.LifeCycleAssessment.Compute.DepletionOfAbioticResourcesFossilFuels(System.Double, System.Double, System.Double)" : "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
"BH.Engine.LifeCycleAssessment.Compute.DepletionOfAbioticResourcesFossilFuels(BH.oM.Dimensional.IElementM, BH.oM.LifeCycleAssessment.MaterialFragments.IEnvironmentalProductDeclarationData)" : "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```

File: BHoMUpgrader42/Upgrades.json [hunk 9]
```diff
@@ -29,10 +29,18 @@
...
+      "BH.Engine.LifeCycleAssessment.Compute.GlobalWarmingPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```
Comment (FraserGreenroyd) [inline BHoMUpgrader42/Upgrades.json:37]:
```suggestion
     "BH.Engine.LifeCycleAssessment.Compute.GlobalWarmingPotential(System.Double, System.Double, System.Double)" : "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
"BH.Engine.LifeCycleAssessment.Compute.GlobalWarmingPotential(BH.oM.Dimensional.IElementM, BH.oM.LifeCycleAssessment.MaterialFragments.IEnvironmentalProductDeclarationData)" : "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```

File: BHoMUpgrader42/Upgrades.json [hunk 10]
```diff
@@ -29,10 +29,18 @@
...
+      "BH.Engine.LifeCycleAssessment.Compute.OzoneDepletionPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```
Comment (FraserGreenroyd) [inline BHoMUpgrader42/Upgrades.json:38]:
```suggestion
      "BH.Engine.LifeCycleAssessment.Compute.OzoneDepletionPotential(System.Double, System.Double, System.Double)" : "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
"BH.Engine.LifeCycleAssessment.Compute.OzoneDepletionPotential(BH.oM.Dimensional.IElementM, BH.oM.LifeCycleAssessment.MaterialFragments.IEnvironmentalProductDeclarationData)" : "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```

File: BHoMUpgrader42/Upgrades.json [hunk 11]
```diff
@@ -29,10 +29,18 @@
...
+      "BH.Engine.LifeCycleAssessment.Compute.PhotochemicalOzoneCreationPotential": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```
Comment (FraserGreenroyd) [inline BHoMUpgrader42/Upgrades.json:39]:
```suggestion
      "BH.Engine.LifeCycleAssessment.Compute.PhotochemicalOzoneCreationPotential(System.Double, System.Double, System.Double)" : "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
"BH.Engine.LifeCycleAssessment.Compute.PhotochemicalOzoneCreationPotential(BH.oM.Dimensional.IElementM, BH.oM.LifeCycleAssessment.MaterialFragments.IEnvironmentalProductDeclarationData)" : "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

File: BHoMUpgrader42/Upgrades.json [hunk 12]
```diff
@@ -29,10 +29,25 @@
 
     }
   },
-  "MessageForDeleted": {
-
-  },
+    "MessageForDeleted": {
+      "BH.Engine.LifeCycleAssessment.Compute.AcidificationPotential(BH.oM.Dimensional.IElementM, BH.oM.LifeCycleAssessment.MaterialFragments.IEnvironmentalProductDeclarationData)": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
+"BH.Engine.LifeCycleAssessment.Compute.AcidificationPotential(System.Double, System.Double, System.Double)", "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```
Comment (FraserGreenroyd) [inline BHoMUpgrader42/Upgrades.json:34]:
```suggestion
"BH.Engine.LifeCycleAssessment.Compute.AcidificationPotential(System.Double, System.Double, System.Double)": "This method has been removed as EvaluateEnvironmentalProductDeclaration contains the same functionality.",
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check versioning

**Comment by FraserGreenroyd** [comment]
@BHoMBot check installer

**Comment by FraserGreenroyd** [comment]
@BHoMBot check core

**Comment by FraserGreenroyd** [comment]
@BHoMBot check copyright-compliance

**Comment by FraserGreenroyd** [comment]
@BHoMBot check core

**Review by FraserGreenroyd — APPROVED** [review]
Functionality was tested by users via a PR artefact. Changes since have been for versioning, so these PRs are fit for merge with passing checks.

**Pull request merged**